librad/src/meta/entity/data.rs

```diff
@@ -35,12 +37,21 @@ pub struct EntitySignatureData {
     pub sig: String,
 }
 
-/// Helper to serialize `HashSet` in a canonical way
-fn ordered_set<S>(value: &HashSet<String>, serializer: S) -> Result<S::Ok, S::Error>
+/// Helper to serialize `HashSet<RadUrn>` in a canonical way
+fn serialize_certifiers<S>(value: &HashSet<RadUrn>, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
Wondering if we could generalise this using `#[serde(from = ..)]` and `#[serde(into =  ..)]` 🤔

Reply:
We would likely need a direct `From <-> Into` connection between `HashSet` and `BTreeSet`.
I fear that the only way to get it would be to newtype HashSet and use the newtype as field type.
I wonder if it's worth...

On a related note, I could not unify those two functions into a generic one because the serde macros would fail.